tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Resources.cs

```diff
@@ -28,6 +28,7 @@ internal static class Resources
         public const string IisMixedRuntimes = "The application pool is configured to host both .NET Framework and .NET Core runtimes. When hosting .NET Core, it's recommended to set '.NET CLR Version' to 'No managed code' to prevent conflict: https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-3.1#create-the-iis-site:~:text=CLR%20version%20to-,No%20Managed%20Code,-%3A";
         public const string OutOfProcess = "Detected ASP.NET Core hosted out of proces. Trying to find the application process.";
         public const string AspNetCoreProcessNotFound = "Could not find the ASP.NET Core applicative process.";
+        public const string AspNetCoreOutOfProcessNotFound = "ASP.NET Core has been detected but no .NET runtime. It could mean that the application is using out-of-process hosting and hasn't received a request to complete the initialization. Please make sure that the site has received at least one web request since the last time it has been restarted.";
```

Comment:
```suggestion
        public const string AspNetCoreOutOfProcessNotFound = "ASP.NET Core has been detected but no .NET runtime. It could mean that the application is using out-of-process hosting and hasn't received a request to complete the initialization. Please make sure that the site has received at least one web request since the last time it was restarted.";
```